src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person;
+  findPerson: (name: string) => Person | null;
+};
+
+let fatherObj: Person | null;
+
+let motherObj: Person | null;
```

Comment:
don't use `let` and don;t place them outside the component. try to fully rewrite the logic